envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/EnvoySnapshotFactoryTest.kt

```diff
@@ -45,6 +45,7 @@ class EnvoySnapshotFactoryTest {
         const val EGRESS_PORT = 3380
         const val CLUSTER_NAME = "cluster-name"
         const val DEFAULT_SERVICE_NAME = "service-name"
+        const val DEFAULT_DISCOVER_SERVICE_NAME = "service-name"
```

Comment:
I'd rather make it unique or reuse DEFAULT_SERVICE_NAME 
```suggestion
        const val DEFAULT_DISCOVER_SERVICE_NAME = "discovery-service-name"
```

Reply:
Also change the constant name to `DEFAULT_DISCOVERY_SERVICE_NAME`.

Reply:
done